kolibri/core/exams/serializers.py

```diff
@@ -100,6 +112,7 @@ def to_internal_value(self, data):
 
     def create(self, validated_data):
         assignees = validated_data.pop('assignments')
+        validated_data['data_model_version'] = 1
```

Comment:
Might be good to associate this value with the Model class, rather than the serializer.

Reply:
I think I understand, but would you mind elaborating on why?

Reply:
Just in case we ever end up creating exam models somewhere else in the code, and so the fact that the current version is tightly coupled to the model itself.